Makefile

```diff
@@ -0,0 +1,56 @@
+# This file was part of Poetry
```

Comment:
Maybe we get just rid of this header? It's MIT licensed anyway.

Reply:
Was actually more meant as a joke :D yeah get rid of it... we don't use licence headers anyway.